test/abstractarray.jl

```diff
@@ -521,3 +521,5 @@ A = TSlowNIndexes(rand(2,2))
 @test_throws ErrorException A[1]
 @test A[1,1] == A.data[1]
 @test first(A) == A.data[1]
+
+@inferred size(rand(3,2,1), 2, 1)
```

Comment:
Maybe also add `@inferred size(rand(3,2,1), 2, 1, 3)` to ensure that you test when engaging a non-empty `Vararg`?